### DISCUSSION TITLE
Add an option to access hidden apps from the home screen

### DISCUSSION THREAD

**Discussion by amin192022** [LawnchairLauncher/lawnchair#3287] [Ideas]
Add an option to access hidden apps from the home screen

![InShot_20230127_185955828](https://user-images.githubusercontent.com/113847974/215164304-9a308730-6ade-4a43-b61f-c7ac6e271c6d.jpg)

**Comment by Abcdefghilkjnm** [comment]
Feels like a good idea until they know that we have something hidden
